### DISCUSSION TITLE
SSH into Kubernetes pod

### DISCUSSION THREAD

**Discussion by MonicaMagoniCom** [rabbitmq/messaging-topology-operator#520] [Q&A] [answered]
Hi all,

I'm new to RabbitMQ messaging topology operator. I've installed it into a Kubernetes cluster and everything seems to work.
However I would like to ssh into the pod to check some variables. Is it possible to ssh into the pod?

Thank you

**Comment by mkuratczyk** [comment, marked as answer]
Do you mean the operator pod or a RabbitMQ server/node pod? For the Operator - there's really not much in there, so it's not very useful. For RabbitMQ pod, it's possible with the usual `kubectl exec -it POD -- bash`.

**Reply by MonicaMagoniCom** [reply]
I mean for the operator pod. So it is not possible... It would be useful to have it!

**Reply by mkuratczyk** [reply]
You can check if [ephemeral containers](https://kubernetes.io/docs/tasks/debug/debug-application/debug-running-pod/#ephemeral-container) are supported in your environment.

Operator's image contains just a few files. In a few years working with Kubernetes, I'm pretty sure I didn't feel the need to ssh into an operator's container (not just RabbitMQ Operator but any operator). What do you expect that to help you with?

**Reply by MonicaMagoniCom** [reply]
For example for Prometheus-operator it is possible to ssh into the pod (but actually, it does not seem to have much in the pod).
Right now, I don't have any special need: I just wanted to check if there were something useful for debugging but you're right, there's not much in operator pods.
